cms/src/index.js

```diff
@@ -143,12 +143,62 @@ module.exports = {
                 },
             });
 
+            const Variables = nexus.inputObjectType({
+                name: "Variables",
+                definition(t) {
+                    t.string("firstName");
+                    t.string("email");
+                },
+            });
```

Comment:
Pokial sa nejedna o classu, tak nevidim dovod, preco by mal byt nazov premennej s velkym zaciatocnym pismenom.  Navyse preco si to nedal rovno do tej definicie a mas to takto mimo este ?

Reply:
Pardon ten uppercase mi usiel, kazdopadne dal som to do premenných nizsie. Mne to prislo ako fajn napad kvoli tomu, aby sme mohli "variables" spreadnut v resolveri.